### ISSUE TITLE
Stats tool no longer dumps output to the terminal window

### ISSUE THREAD

**Issue by IanHeywood** [ska-sa/tigger#122]
Old Tigger used to print the region stats that pop up to the terminal, which was very handy for copying and pasting positions and measurements.

So I guess this is a feature request!

**Comment by bennahugo** [comment]
It is now automatically added to the clip board. So I think we can close
this?

**Comment by IanHeywood** [comment]
Can we have both please?

The shift+click single position measurement still prints to the terminal.

**Comment by o-smirnov** [comment]
I agree printing is more useful than just automatic clipboard copy. I have often printed several, then indulged in a cut-and-paste buffet.

**Comment by razman786** [comment]
This is fixed in PR https://github.com/razman786/tigger/pull/6. Whatever is sent to the clipboard is now also sent to the terminal.
